momento-sdk/src/main/java/momento/sdk/CacheClient.java

```diff
@@ -506,6 +507,48 @@ public CompletableFuture<CacheListConcatenateBackResponse> listConcatenateBack(
     return scsDataClient.listConcatenateBack(cacheName, listName, values, ttl, truncateFrontToSize);
   }
 
+  /**
+   * Concatenates values to the front of the list.
+   *
+   * @param cacheName Name of the cache to store the item in
+   * @param listName The list in which the value is to be added.
+   * @param values The elements to add to the list.
+   * @param ttl Time to Live for the item in Cache. This ttl takes precedence over the TTL used when
+   *     building a cache client {@link CacheClient#builder(String, Duration)}
+   * @param truncateBackToSize If the list exceeds this length, remove excess from the front of the
+   *     list. Must be positive.
+   * @return Future containing the result of the list concatenate back operation.
+   */
+  public CompletableFuture<CacheListConcatenateFrontResponse> listConcatenateFront(
+      String cacheName,
+      String listName,
+      List<String> values,
+      CollectionTtl ttl,
+      Integer truncateBackToSize) {
```

Comment:
Why is this an Integer instead of an int?

Reply:
I didn't realize I was using Integer here. That was me experimenting before in the list concatenate back api. Let me correct both and push out another rev